cdk/lib/security-hq.ts

```diff
@@ -124,9 +125,41 @@ dpkg -i /tmp/installer.deb`,
           new GuDynamoDBWritePolicy(this, 'DynamoWrite', {
             tableName: cfnTable.tableName as string,
           }),
+          new GuAllowPolicy(this, 'SsmCommands', {
+            resources: ['*'],
+            actions: [
+              'ec2messages:AcknowledgeMessage',
+              'ec2messages:DeleteMessage',
+              'ec2messages:FailMessage',
+              'ec2messages:GetEndpoint',
+              'ec2messages:GetMessages',
+              'ec2messages:SendReply',
+              'ssm:UpdateInstanceInformation',
+              'ssm:ListInstanceAssociations',
+              'ssm:DescribeInstanceProperties',
+              'ssm:DescribeDocumentParameters',
+              'ssmmessages:CreateControlChannel',
+              'ssmmessages:CreateDataChannel',
+              'ssmmessages:OpenControlChannel',
+              'ssmmessages:OpenDataChannel',
+            ],
+          }),
```

Comment:
This looks the same as the [`GuSSMRunCommandPolicy` construct](https://github.com/guardian/cdk/blob/624f45762ec875a905c7d8759166ce0f20e747fd/src/constructs/iam/policies/ssm.ts#L5). Can it be used?

Reply:
Ooh possibly. Will add.